static/app/views/settings/account/accountSubscriptions.tsx

```diff
@@ -40,12 +41,13 @@ class AccountSubscriptions extends AsyncView<AsyncView['props'], State> {
     return 'Subscriptions';
   }
 
-  handleToggle = (subscription: Subscription, index: number, _e: React.MouseEvent) => {
+  handleToggle = (subscription: Subscription, listId: number, _e: React.MouseEvent) => {
     const subscribed = !subscription.subscribed;
     const oldSubscriptions = this.state.subscriptions;
 
     this.setState(state => {
       const newSubscriptions = state.subscriptions.slice();
+      const index = findIndex(newSubscriptions, {listId});
```

Comment:
Ah is this looking up like `[{listId: 1, ...}, {listId: 3, ...}]` type thing?

Reply:
@evanpurkhiser yep, it's matching off the list id